backend/backend/src/main/java/de/aaaaaaah/velcom/backend/access/TaskWriteAccess.java

```diff
@@ -71,6 +72,8 @@ public void onTaskDelete(Consumer<TaskId> deleteHandler) {
 	public Collection<Task> insertTasks(Collection<Task> tasks) {
 		try (DBWriteAccess db = databaseStorage.acquireWriteAccess()) {
 			return insertTasks(tasks, db);
+		} catch (DataAccessException e) {
+			return List.of();
```

Comment:
This sounds a bit broad? Every DB error is just silently swallowed, without even being logged? If the database dies or any other syntax/semantic error occurs, the user would just think it already existed and no logs exist to trace it down.

Maybe make this a bit more specific? If I misunderstood when the DataAccessException is thrown, this comment is invalid.